linkerd/stack/src/shared.rs

```diff
@@ -2,14 +2,16 @@ use futures::Poll;
 use linkerd2_error::Never;
 use tower_service as svc;
 
-pub fn shared<V: Clone>(v: V) -> Shared<V> {
-    Shared(v)
-}
-
 /// Implements `Service<T>` for any `T` by cloning a `V`-typed value.
 #[derive(Clone, Debug)]
 pub struct Shared<V>(V);
 
+impl<V: Clone> Shared<V> {
+    pub fn new(v: V) -> Self {
```

Comment:
Is there a reason this was changed from a free fn to a `new`?

Reply:
Probably not, other than adopting tower idioms.